database_adapters/db_adapters.py

```diff
@@ -79,7 +81,10 @@ def get_all_collection_data(self, database_name, collection_name):
         collection = self.client[database_name][collection_name]
         result = collection.find()
         result = loads(json.dumps(list(result), default=str))
-        return result
+        size_per_item = sys.getsizeof(result) / len(result)
+        items = int(settings.MAX_DOWNLOADED_SIZE // size_per_item)
+        return result[:items]
+
```

Comment:
Instead of this, why we just put an if in job_data view to return a message indicating the max downloaded size.

Reply:
Instead of return the limited jod_data it should return just a message ?